src/index.ts

```diff
@@ -588,16 +609,42 @@ export class Bigtable {
     );
   }
 
+  getInstances(callback: GetInstancesCallback): void;
+  getInstances(
+    gaxOptions: GetInstancesRequest,
+    callback: GetInstancesCallback
+  ): void;
+  getInstances(gaxOptions?: GetInstancesRequest): Promise<GetInstancesResponse>;
+  /**
+   * Query object for listing instances.
+   *
+   * @typedef {object} GetInstancesRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetInstancesResponse
+   * @property {Instance[]} 0 Array of {@link Instance} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * @callback GetInstancesCallback
+   * @param {?Error} err Request error, if any.
+   * @param {Instance[]} instances Array of {@link Instance} instances.
+   * @param {object} apiResponse The full API response.
+   */
   /**
    * Get Instance objects for all of your Cloud Bigtable instances.
    *
-   * @param {object} [gaxOptions] Request configuration options, outlined here:
+   * @param {GetInstancesRequest} [gaxOptions] Request configuration options, outlined here:
```

Comment:
This should probably be `object` again?
Also I believe we should fix the link below to 
https://googleapis.github.io/gax-nodejs/classes/CallSettings.html